### DISCUSSION TITLE
Let's set Oqtane up for sponsorship!

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#4362] [Ideas]
I was reading this article about how to attract sponsorship, and I feel Oqtane could really benefit from turning in donations.  

https://opensource.microsoft.com/blog/2024/06/27/5-things-we-learned-from-sponsoring-a-sampling-of-our-open-source-dependencies/

**Comment by sbwalker** [comment]
@markdav-is I have been involved in maintaining Open Source for more than 20 years and based on my experiences, sponsorship and donations is not a viable funding model. I actually did a presentation on this topic at the .NET Foundation Summit last year https://youtu.be/VYi7GggFyxc?t=12880

**Comment by hishamco** [comment]
> on my experiences, sponsorship and donations is not a viable funding model

Agree, but I noticed some OSS projects got a great fund either in GitHub or OpenCollective especially if the core team active
